test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+﻿Describe "ConvertTo-Html DRT Unit Tests" -Tags "CI" {
```

Comment:
We are trying to move away from the Windows BVT/DRT terminology.
Please, just use the descriptive names for the `Describe`. More testing guidelines are here: https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#writing-pester-tests